packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -64,6 +64,24 @@ export const SetupFlopFlipProvider = (props: Props) => {
     [flags, props.ldClientSideId, props.projectKey, props.user]
   );
 
+  if (process.env.NODE_ENV === 'test') {
+    const memoryAdapter = require('@flopflip/memory-adapter').default;
```

Comment:
Not sure if this the "best" way to do that, but we need to use the memory adapter when testing the app shell.
Previously we were kind of mocking the requests to launchdarkly, but that is a bit tricky and error prone.

Reply:
Yeah, makes total sense. It's not possible to just wrap it in a `flopflip-adapter.js` which does export either or them based on the env?

Reply:
I double checked the rollup + webpack bundle, and it's correctly stripped from the webpack bundle:

<img width="2077" alt="image" src="https://user-images.githubusercontent.com/1110551/91160782-206cb980-e6ca-11ea-9b55-decad3387359.png">